pkg/api/mlflow/dao/repositories/run.go

```diff
@@ -292,6 +292,12 @@ func (r RunRepository) UpdateWithTransaction(ctx context.Context, tx *gorm.DB, r
 	if err := tx.WithContext(ctx).Model(&run).Updates(run).Error; err != nil {
 		return eris.Wrapf(err, "error updating existing run with id: %s", run.ID)
 	}
+	// Use UpdateColumn so we can reset EndTime to null
+	if run.EndTime.Int64 == 0 {
+		if err := tx.Model(&run).UpdateColumn("EndTime", sql.NullInt64{}).Error; err != nil {
```

Comment:
we specially use `UpdateColumn` to just make field nullable, right?

Reply:
Exactly, regular `Updates` function skip nil values

Reply:
even if you pass like `sql.NullInt64{IsValid: true}`?

Reply:
It actually works! Thanks!

Reply:
ohh, that's nice! thank you!